pydis_site/apps/events/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, re_path
+
+from pydis_site.apps.events.views import IndexView, PageView
+
+app_name = "events"
+urlpatterns = [
+    path("", IndexView.as_view(), name="events"),
```

Comment:
```suggestion
    path("", IndexView.as_view(), name="index"),
```
I touched on this one in the other PR, but using `{% url 'events:index' %}` is more explicit than `{% url 'events:events'%}`.

Reply:
If you're actioning this, replace [these lines](https://github.com/python-discord/site/pull/417/files#diff-2fc0bf29e735c0301788f0619f6b28d3c2d43074a1f6cfe2a9a273eb64de2267R6-R7) in the index template with the url tag equivalents, e.g. `{% url 'events:index' %}`